src/osd/pg_scrubber.cc

```diff
@@ -2004,6 +2025,30 @@ void PgScrubber::reset_internal_state()
   m_active = false;
 }
 
+// note that only applicable to the Replica:
+void PgScrubber::advance_token()
+{
+  dout(10) << __func__ << " was: " << m_current_token << dendl;
+  m_current_token++;
```

Comment:
How does this ensure that it matches the primary?  Is token strictly local to the replica?